src/index.html

```diff
@@ -13,6 +13,57 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--yellow">
```

Comment:
There is no need for a blank line between the <body> and the first <div> element. Blank lines should not be between parent and child elements.